tests/Integration/Behaviour/Features/Context/Domain/CartFeatureContext.php

```diff
@@ -231,6 +238,45 @@ public function addCustomizedProductToCarts(int $quantity, $productReference, $r
         );
     }
 
+    /**
+     * @When I add :quantity customized products without its customization with reference :productReference to the cart :reference
+     */
+    public function addCustomizedProductToCartsWithoutCustomization(
+        int $quantity,
+        string $productReference,
+        string $reference
+    ) {
+        $productId = (int) Product::getIdByReference($productReference);
+        $product = new Product($productId);
+        $customizationsFields = $product->getCustomizationFields();
+        if (empty($customizationsFields)) {
+            throw new Exception('The product has no customizables fields');
+        }
+        $cartId = (int) SharedStorage::getStorage()->get($reference);
+
+        try {
+            $this->getCommandBus()->handle(
+                new UpdateProductQuantityInCartCommand(
+                    $cartId,
+                    $productId,
+                    $quantity,
+                    null,
```

Comment:
Maybe you should use empty `array` rather than `null` ?

Reply:
No Type is `int|null`